### PR TITLE
Camunda Modeler v4.0.0-alpha.3 blog post

### PR DESCRIPTION
**Pull request by oguzeroglu**

### CODE REVIEW

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+We are excited to announce the release of Camunda Modeler v4.0.0-alpha.3. This release introduces the new Camunda Modeler icon and error tracking features.
+
+[Download the latest release](https://downloads.camunda.cloud/release/camunda-modeler/4.0.0-alpha.3/) and start modeling right away.
+
+<!--more-->
+
+## New Camunda Modeler Icon
+
+As a part of Camunda rebranding, we are saying goodbye to our old Camunda Modeler icon and introducing the new icon with v4.0.0-alpha.3. You may see this icon in your taskbar, app launch menu, etc.
```
Comment (pinussilvestrus) [inline content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:17]: 'icon' is used twice in the same sentence. Maybe

> As a part of Camunda rebranding, we are saying goodbye to our old Camunda Modeler app icon and introducing the new one with this release. You may see this icon in your taskbar, app launch menu, etc.

Comment (pinussilvestrus) [reply content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:17]: Also maybe we should clarify why we did that, e.g. due to the Camunda rebranding

https://blog.camunda.com/post/2020/01/something-bold-something-new-fresh-look/

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+[Please let us know](https://forum.camunda.org/c/modeler) how you feel about the new icon.
+
+
+{{< figure class="no-border teaser" src="icon.png" alt="The new Camunda icon" caption="The new Camunda Modeler icon" >}}
+
+## Error Tracking
+
+Camunda Modeler v4.0.0-alpha.3 is shipped with error tracking feature for those who enabled the `Enable Error Reports` option in Privacy Preferences. Please keep in mind that you can always turn on/off this feature by going to `Help>Privacy Preferences`.
```
Comment (pinussilvestrus) [inline content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:26]: Maybe
> with **an** error tracking feature

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md (hunk 1 above)
Comment (pinussilvestrus) [reply content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:17]: Ignore the second comment ;) Somehow missed the first part of the sentence.

Comment (oguzeroglu) [reply content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:17]: I think it makes sense though. I'll convert the sentence into a link pointing to our rebranding blog post.

**Comment by oguzeroglu** [comment]
Thanks! Addressed the issues you 👉

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+We are excited to announce the release of Camunda Modeler v4.0.0-alpha.3. This release introduces the new Camunda Modeler icon and error tracking features.
+
+[Download the latest release](https://downloads.camunda.cloud/release/camunda-modeler/4.0.0-alpha.3/) and start modeling right away.
+
+<!--more-->
+
+## New Camunda Modeler Icon
+
+[As a part of Camunda rebranding](https://blog.camunda.com/post/2020/01/something-bold-something-new-fresh-look/), we are saying goodbye to our old Camunda Modeler icon and introducing the new one with v4.0.0-alpha.3. You may see this icon in your taskbar, app launch menu, etc.
```
Comment (philippfromme) [inline content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:17]: How about:

As part of _the recently announced_ Camunda rebranding we _say_ goodbye to _the_ old Camunda Modeler icon and _introduce a_ new _one as part of this alpha release._

Just a suggestion.

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+[Please let us know](https://forum.camunda.org/c/modeler) how you feel about the new icon.
+
+
+{{< figure class="no-border teaser" src="icon.png" alt="The new Camunda icon" caption="The new Camunda Modeler icon" >}}
+
+## Error Tracking
+
+Camunda Modeler v4.0.0-alpha.3 is shipped with an error tracking feature for those who enabled the `Enable Error Reports` option in Privacy Preferences. Please keep in mind that you can always turn on/off this feature by going to `Help>Privacy Preferences`.
```
Comment (philippfromme) [inline content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:26]: > _This alpha release ships_ with an error tracking feature

> in _the_ Privacy Preferences.

> You can always turn on/off this feature by going to `Help>Privacy Preferences`.

I'd remove the _Please keep in mind_, it sounds like a warning.

Comment (philippfromme) [reply content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:26]: Some suggestions. I wouldn't say Camunda Modeler v4.0.0-alpha.3 too often, I think it's enough to refer to it as the alpha release. People should know what version this after reading the first part of the blog post.

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+{{< figure class="no-border teaser" src="error-tracking.png" alt="Enabling Error Reports " caption="Error Reports option in Privacy Preferences dialog" >}}
+
+If you want to learn more about what kind of information we collect and why we collect it, read our [Privacy Policy](https://camunda.com/legal/privacy/).
+
+## Looking Forward
+
+Read the [changelog](https://github.com/camunda/camunda-modeler/blob/master/CHANGELOG.md#400-alpha3) to learn about all features, bug fixes, and numerous additional improvements that made it into this release. [Download the new modeler](https://downloads.camunda.cloud/release/camunda-modeler/4.0.0-alpha.3/), give it a try, and, most importantly, give us feedback.
```
Comment (philippfromme) [inline content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:34]: learn about all _the_ features

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md (hunk 3 above)
Comment (nikku) [reply content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:17]: I like both versions.

File: content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md (hunk 4 above)
Comment (nikku) [reply content/post/2020/05/camunda-modeler-4.0.0-alpha.3-released.md:26]: Agreed to @philippfromme

**Comment by oguzeroglu** [comment]
Thanks for the feedback guys. I addressed the issues you mentioned. Let get this merged?

**Pull request merged**